taxonomies/labels.txt

```diff
@@ -13,7 +13,7 @@ stopwords:en:ingredients, agricultural, from, with, of, by, the, verified, certi
 stopwords:bg:съставки, земеделие, от, с, контролиран, сертифициран, 100%, натурален, гарантиран, продукт
 stopwords:da:fra, kontrolleret, dyrkning, dyrket, dyrkede, råvarer, landbrug, certificeret
 stopwords:de:Zutaten, von, mit, enthält, kontrolliert, zertifiziert, geprüft, 100%, rein, Produkt, Garantie, garantiert, Standard, während
-stopwords:es:por, la, las, los, el, apto, para, verificado, certificado, aprovado, controlado, registrado, 100%, garantía, garantizado, garantizada, estándar
+stopwords:es:por, la, las, los, el, apto, para, con, de, verificado, certificado, aprovado, controlado, registrado, 100%, garantía, garantizado, garantizada, estándar, procedente, libre
```

Comment:
> We shouldn't put "libre" here, or it would make "Libre de trigo" a synonym of "Con trigo".
> 
Oh, maybe I didnt understood how the stopwords work then... thanks!

> What about "procedente", is it useful to match some label synonyms?

Many times the ingredients say "product 1*, product 2*, product 3. *procedente de agricultura ecológica.